example.yaml

```diff
@@ -34,36 +34,28 @@ sops:
-    pgp:
-    -   created_at: '2019-12-10T22:45:53Z'
-        enc: |-
-            -----BEGIN PGP MESSAGE-----
-
-            wcBMAyUpShfNkFB/AQgAE0MaWAQGbTKY7Xg3fDNtzlvnVBkkQRHsLt5kUTu2nAy4
-            sPX0NRXPVF/tAMxr9mI2fRjKnNBXKpOAecNis85D/QEkfflG8/syGkqiJqy9Nqon
-            WSm1bNriPfD4PL850688EJe49Xrsz6rVbW5FYZCHMbnPxvmoheMJRxLonW3/eWPy
-            IjJ9i6Z7W175mv1y7FELOimdQeelynp4r8bOuuq1BhePB4+wJXihw9n0ovuLklpl
-            Kr2iCmIUibSywlEO/LGQT/VXo6R7xgSN2Xg0RwWfflajYHNhVkHlnNYzkACjvdhj
-            ph1M5fLGDqPu+ySSe93EyahNhdwKgQ7R9yF8/13+QdLgAeRmrBgh0N54mQQAQyL7
-            mH/i4SWi4BjgmeHgUeBL4qcm2avg6OUasVJQHlTeA8D+c3TwKSTRVDijN4GBadYJ
-            Z5/vXKOseeBk5JWCnIHC/MtjOkuPt53nvGzi3lYvW+F0FQA=
-            =RDyn
-            -----END PGP MESSAGE-----
-        fp: FBC7B9E2A4F9289AC0C1D4843D16CEE4A27381B4
-    -   created_at: '2019-12-10T22:45:53Z'
-        enc: |-
-            -----BEGIN PGP MESSAGE-----
-
-            wYwDXFUltYFwV4MBBABpm+tFhFhv3A7A/L/p6nL3HXKKhONrgguYgXA/hhSg4/bD
-            1Po5pQhCM4yb3gqWewxgVpGNKFr/Gl+kN9eZ3LXp5nEdhei/aQn7BbWkhph5PKt6
-            faiEZAL5PNHvktvEQwPsfNJvxe8QT2Z9oFmlueP0n3mCZz3UV9LZHNwOP7XfzdLg
-            AeRzUGrZK43KavmIjdgPXcd/4ZW14NPgNeGm8+BO4nD26Fvgh+UiCb0TpBiI7WsX
-            HQGUlFjuR6Vd7Q+vg8B/1Ovm3fUw7uCG5Gc2WeB0M+pXaLYS9bCDL9jisGOkD+Ef
-            ZAA=
-            =mqGc
-            -----END PGP MESSAGE-----
-        fp: D7229043384BCC60326C6FB9D8720D957C3D3074
+    hc_vault: []
+    age:
+        - recipient: age1yt3tfqlfrwdwx0z0ynwplcr6qxcxfaqycuprpmy89nr83ltx74tqdpszlw
+          enc: |
+            -----BEGIN AGE ENCRYPTED FILE-----
+            YWdlLWVuY3J5cHRpb24ub3JnL3YxCi0+IFgyNTUxOSBnYU4zaDZ3dFNXTGRLN2Qr
+            ZFdhNGFGUHN2WGxYb1FSNFN6Tk02cHZIMkRnCkVCWXh2NVlTUkRQME9TeGZaaElq
+            eTRieEI1WGdTK1pwL1c1eGtKTDFBUW8KLS0tIFNQeFZSbnJ4QWx3K0wwOFJZODUy
+            SDQ0NDhjcVpYMjYxcm1ZdXFYaWtkakUKWgQTWzf+7rgMjC9rPLSy6jTGbI71hMnC
+            MuRgd5LreYBQo/hRZUYuQ4jmGgOnn3GaUZE1HpSqjjaX8jYn4lNqJA==
+            -----END AGE ENCRYPTED FILE-----
+        - recipient: age1tmaae3ld5vpevmsh5yacsauzx8jetg300mpvc4ugp5zr5l6ssq9sla97ep
+          enc: |
+            -----BEGIN AGE ENCRYPTED FILE-----
+            YWdlLWVuY3J5cHRpb24ub3JnL3YxCi0+IFgyNTUxOSBhRkUwZ2RKMzZobERMcFdF
+            NGhyNVhlNnNNbUh0NzhMaEgxSlpTL2xRWG4wCkZPdWZsY01HbUUremxBdTRwZ0Nq
+            by85NFZWRTIvazVsZmxIRTByanNUVkEKLS0tIEE0c2h5RHB3c0JHSkhuSU5hTjlM
+            SnJJeGxoQXJ4SlY3d3J4U3Q3cEo2UjQKpI7iyx0ENFxlGnsM7ymneO1YUU5xgo88
+            VRjAc4Tb+mKk+qIeY6rY2AfsKZ8HQsA/CaejRmWT00iXUYj1S5eBww==
+            -----END AGE ENCRYPTED FILE-----
+    lastmodified: "2022-03-15T22:11:15Z"
+    mac: ENC[AES256_GCM,data:dwrqrNqt0F0lNkPv/T16GzC57nGmHo/frkdYvsUM/Izfqd7F7t1FUs6sILmz1o6lKllkADL3+A5DISSsaYK05RYr2RuCtZ7V+XC1JKD+loJoo1CAwcHAjGkv/Jv2Y7ePd6tXr6Fb+BuFVQIpwmI5ze3DsKqUd7bQ+DfRRpr9IoA=,iv:9Y2ASpFstcE8ZgwzOvb3oMQsd30Rukd7gvOb7lMvV1o=,tag:qXQosmzw6UoOXFI+8IvtXg==,type:str]
+    pgp: []
```

Comment:
Why is `pgp` empty when there are still pgp keys in the `.sops.yaml`? I'm guessing they just weren't used, but we should keep them or remove the `pgp` keys in the `.sops.yaml`.